src/datasources/balances-api/balances-api.manager.spec.ts

```diff
@@ -16,48 +16,71 @@ const valkBalancesApi = {
 
 const valkBalancesApiMock = jest.mocked(valkBalancesApi);
 
+const zerionBalancesApi = {
+  getBalances: jest.fn(),
+  clearBalances: jest.fn(),
+} as IBalancesApi;
+
+const zerionBalancesApiMock = jest.mocked(zerionBalancesApi);
+
 describe('Balances API Manager Tests', () => {
   describe('useExternalApi checks', () => {
     configurationServiceMock.getOrThrow.mockImplementation((key) => {
       if (key === 'features.valkBalancesChainIds') return ['1', '2', '3'];
+      if (key === 'features.zerionBalancesChainIds') return ['4', '5', '6'];
```

Comment:
If a chain is present in both collections then it'd return the Valk provider right? This would be a configuration issue so not that relevant given that this is a testing phase.

Reply:
Yes, I also thought about that. Probably the most correct solution would be to check at bootstrap time that the provider's configured chains don't overlap, but given this is a testing phase, I thought it would add too much effort/complexity for a low benefit.

In my opinion that would be the way to solve the issue if several providers would be a permanent scenario.

Reply:
> but given this is a testing phase, I thought it would add too much effort/complexity for a low benefit.

Yes exactly 👍 No action needed here it was mostly to clarify the setup.